backend/testUtils/class.ts

```diff
@@ -110,7 +105,6 @@ export const assertResponseMatchesExpected = (
   expect(result.schoolYear).toEqual(expected.schoolYear);
   expect(result.gradeLevel.toString).toEqual(expected.gradeLevel.toString);
   expect(result.teacher).toEqual(mockTeacher);
-  expect(result.testSessions).toEqual([mockTestSessionWithId]);
```

Comment:
i think we still need this check. i would suggest you create new constants with the expected values for each call

Reply:
I just added the assertion to assert that `result.testSession` is equal to `[]`, is this okay? None of the tests acutally update the class to hold test sessions, since `createClass` and `updateClass` take in ClassRequestDTOs which no longer carry the `testSessions` field, I am assuming in the future there will be additional functions to add testSessions which will then be more properly tested, like `createStudent`, `update_student` and such.

Reply:
wait but `getClassById` for example will still return test sessions, so that unit test should use a mockClass that has test sessions. whereas for create and edit, the mock value used will not have that field

Reply:
Ah yes this is true - I made a change to assert that specifically in getClassById test, let me know if that works

Reply:
hmm, sorry for being picky here.. but just to avoid more refactoring in the future since we'll be adding new functions that do have test sessions i think it might be better to directly use the `assertResponseMatchesExpected` function. i believe you can still do comparison if the value is undefined - if not, you can add a check if expected.testSessions is defined

Reply:
Oh whoops I am just seeing this now - I made the change, let me know if this works